_source/logzio_collections/_tracing-sources/dotnet-framework-otel-auto.md

```diff
@@ -285,18 +285,31 @@ You can use the following options to update the Helm chart parameters:
 
 ###### Example
 
+You can run the logzio-k8s-telemetry chart with your custom configuration file that takes precedence over the `values.yaml` of the chart.
+
 ```
-helm install logzio-otel-traces logzio-helm/logzio-otel-traces -f my_values.yaml 
+helm install logzio-k8s-telemetry logzio-helm/logzio-k8s-telemetry -f <PATH-TO>/my_values.yaml 
 ```
 
+Replace `<PATH-TO>` with the path to your custom `values.yaml` file.
+
+###### Optional parameters
+
+If required, you can add the following optional parameters as environment variables:
+  
+| Parameter | Description | 
+|---|---|
+| secrets.SamplingLatency | Threshold for the spand latency - all traces slower than the threshold value will be filtered in. Default 1000. | 
```

Comment:
That's not the default value (0)
https://github.com/logzio/logzio-helm/blob/master/charts/logzio-telemetry/values.yaml#L25